<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullShowTest</class>
 <widget class="QMainWindow" name="FullShowTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FullShowTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="WndImage" name="id_full_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>531</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="id_btn_full_show">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>全屏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="id_btn_normal">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复</string>
    </property>
   </widget>
   <widget class="QWidget" name="id_widget" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WndImage</class>
   <extends>QWidget</extends>
   <header>wndimage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
